common/test/test_migrations/utils.py

```diff
@@ -0,0 +1,65 @@
+"""
+General class for Test migrations.For more information visit at
```

Comment:
also, I think you can say, instead of "For more information, visit", something like "based off an implementation provided in"